spec/helpers/state_file/nj_tenant_eligibility_helper_spec.rb

```diff
@@ -22,22 +22,6 @@
       end
     end
 
-    describe "states that require applicant to use the worksheet" do
-      context "when YES to tenant_more_than_one_main_home_in_nj" do
-        let(:intake) { create :state_file_nj_intake, tenant_more_than_one_main_home_in_nj: "yes" }
-        it "returns worksheet required" do
-          expect(described_class.determine_eligibility(intake)).to eq(described_class::WORKSHEET)
-        end
-      end
-
-      context "when YES to tenant_shared_rent_not_spouse" do
-        let(:intake) {  create :state_file_nj_intake, tenant_shared_rent_not_spouse: "yes" }
-        it "returns worksheet required" do
-          expect(described_class.determine_eligibility(intake)).to eq(described_class::WORKSHEET)
-        end
-      end
-    end
-
```

Comment:
[pebble/boulder if you want re-review] GitHub isn't letting me select the right lines of code to put this comment on, but this whole file should be reviewed & revised. Several of the other tests in here (like the "advance state" tests) reference the `tenant_shared_rent_not_spouse` and `tenant_more_than_one_main_home_in_nj` checkboxes, and since those are no longer part of the code for this helper component, they should be cleaned up in the tests as well